index.html

```diff
@@ -0,0 +1,28 @@
+
+<head>
+  <link rel="stylesheet" href="style.css">
+  <title>Welcome to our page</title>
+</head>
+
+<body>
+  <nav id="navbar">
+    <a href="#projects" target="bottom">Go to projects</a> 
+    <form action="https://www.google.com/" target="blanck"><input type="search" placeholder="Search here" ><input type="submit" value="submit"></nav></form>
+    
```

Comment:
- [ ] I think we can improve this comment review message by being more polite as it sounds like an order. I suggest reforming your message politely by proceeding like this:
> "I think it would be better to..." 

Or similar to this :point_up:

Reply:
I have already modified my comment, thank you very much!  @tresorsawasawa